src/components/algolia-search/components/panel.tsx

```diff
@@ -110,12 +117,11 @@ export default forwardRef(function Panel<THit extends Hit<unknown>>(
         style={panelPositionStyle}
       >
         <div className={s.panelLayout}>
-          {collections.map((collection, index) => {
-            const { source, items } = collection
-
-            return (
-              <section key={`source-${index}`}>
-                {items.length > 0 && (
+          {showResults ? (
```

Comment:
Adjusted this so it will work if / when we have multiple collections. Previously it was just assuming there was only one collection.